roles/installer/templates/jobs/migration.yaml.j2

```diff
@@ -28,7 +28,7 @@ spec:
           volumeMounts:
             - name: "ca-trust-extracted"
               mountPath: "/etc/pki/ca-trust/extracted"
-            - name: "{{ ansible_operator_meta.name }}-bundle-cacert"
+            - name: "{{ bundle_cacert_secret }}"
```

Comment:
Renaming this mount makes it less consistent with the other operators, eda-server-operator for example.  Do we want to rename this everywhere?